README.md

```diff
@@ -49,7 +49,8 @@ See [Troubleshooting Tips](doc/troubleshooting.md) as well as [Yarn tips](doc/ya
 | 0.5.0       | 1.2.0         |
 | 0.5.1       | 1.3.0         |
 | 0.5.2       | 1.3.1         |
-| master      | 1.4.1         |
+| 0.5.3       | 1.4.1         |
```

Comment:
Did we release 0.5.3 yet? Does this mean that there will not be a release that supports 1.4.1? Or will master be compatible with 1.4.1 too?